java/arcs/core/storage/DirectStoreMuxer.kt

```diff
@@ -87,7 +87,8 @@ class DirectStoreMuxer<Data : CrdtData, Op : CrdtOperationAtTime, T>(
   /**
    * Gets data from the store corresponding to the given [referenceId].
    */
-  suspend fun getLocalData(referenceId: String) = store(referenceId).store.getLocalData()
+  suspend fun getLocalData(referenceId: String, callbackId: Int) =
```

Comment:
What purpose does the callbackId serve here? This would be good context to add to the KDoc, IMO.

Reply:
Nothing yet.. in the next PR I will make use of the callbackId. I made this a separate PR because it's little but affects 6 files so I wanted to not get that muddled in with the larger change that mostly affects the DSM. Do you think I should explain in the KDoc what it _will_ be used for and just mention it hasn't been implemented yet?

Here's the context for what it will be needed for: At the moment, the reference mode store assumes that when it calls `backingStore.getLocalData(refId)`, it is now listening to any changes from that direct store of that particular `refId`. That is the case for now because there is a single communication channel between the DirectStoreMuxer and a DirectStore, and any updates from the DirectStore just gets broadcast to the ReferenceModeStore. 

However, I'm about to make some changes where there will be one DirectStoreMuxer-DirectStore communication channel per (StorageProxyMuxer or ReferenceModeStore)-DirectStoreMuxer communication channel. Basically what this means is when the reference mode store calls `backingStore.getLocalData(refId, callbackId)`, the DirectStoreMuxer will need to make sure there is a corresponding communication channel between the DirectStoreMuxer and the DirectStore for that callbackId so that updates from the DirectStore will be sent to the ReferenceModeStore (This is the thing that will be implemented in the next PR).

Reply:
Great, thanks for the details :)